client-tags/language.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Language client tag
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Val Lorentz"
+    period: 2020
+    email: "progval+ircv3@progval.net"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `+language` tag name. Instead, implementations SHOULD use the
+`+draft/language` tag name to be interoperable with other software
+implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed tag name.
+
+## Introduction
+
+This specification defines a client-only message tag to indicate the language
+of a message.
+
+## Architecture
+
+### Dependencies
+
+Clients wishing to use this tag MUST negotiate the
+[`message-tags`](../extensions/message-tags.html) capability with the server.
+
+### Format
+
+The language tag is sent by a client with the client-only prefix `+` and
+its value must be a a ISO 639-3 code.
```

Comment:
No difference between e.g. British and American English?

Reply:
No. Even variants of arabics are considered the same macrolanguage.

Reply:
There's https://tools.ietf.org/html/bcp47 which is the commonly used `en-US` `en-GB` `en` `fr` etc